PyStemmusScope/iostreamer.py

```diff
@@ -0,0 +1,149 @@
+"""PyStemmusScope directories utilities.
+
+Module designed to manage input directories and data for running the model
+and storing outputs.
+"""
+import distutils.dir_util
+import distutils.file_util
+import os
+import time
+from pathlib import Path
+
+
+class InputDir:
+    """Create input directories and copy required data."""
+
+    def __init__(self, path_to_config_file: str = "config_file_snellius.txt",
+        path_to_model: str = "path_to_STEMMUS_SCOPE_repository"
+    ):
+        """Instantiate a handler for work directories.
+
+        Specify path STEMMUS SCOPE repository and load the config file.
+
+        Args:
+            path_to_config_file: Path to the config file.
+            path_to_model: Path to the model.
+        """
+    
+        self.config = self._read_config(path_to_config_file)
+        self.path_to_model = path_to_model
+
+    def __str__(self):
+        return "Input directories handler."
+
+    def _read_config(self, path_to_config_file):
+        """Read config from given config file.
+
+        Load paths from config file and save them into dict.
+
+        Args:
+            path_to_config_file: Path to the config file.
+
+        Returns:
+            Dictionary containing paths to work directory and all sub-directories.
+        """
+        config = {}
+        with open(path_to_config_file, "r", encoding="utf8") as f:
+            for line in f:
+                (key, val) = line.split("=")
+                config[key] = val.rstrip('\n')
+
+        return config
+
+    def prepare_work_dir(self, forcing_filenames_list: list,
+        full_run: bool = False
+    ):
+        """Prepare work directory for each station.
+
+        Prepare work directory based on the given list of forcing. This function
+        creates directory for each forcing data and name it using the name of the
+        station.
+
+        Args:
+            forcing_filenames_list: List of forcing filenames to be used for the experiments.
+            full_run: If true, all stations with forcing listed in the ForcingPath will be used.
+
+        Returns:
+            Dictionary containing path to work directory and config file for every station/forcing.
+        """
+        self.forcing_filenames_list = forcing_filenames_list
+        
+        if full_run:
+            self.forcing_filenames_list = [file.name for file in Path(self.config["ForcingPath"]).iterdir()]
+
+        # dict to store path to config file and work directory for each station
+        self.work_dir_dict, self.config_path_dict = self._create_work_dir()
+
+        return self.work_dir_dict, self.config_path_dict
+
+    def _create_work_dir(self):
```

Comment:
```suggestion
    def create_io_dir(self):
```

Reply:
let's rename it because it actually creates `input_dir` and `output_dir` under the existing work directory.

Reply:
It is renamed now.